lib/cartopy/tests/mpl/test_crs.py

```diff
@@ -7,12 +7,12 @@
 import pytest
 
 import cartopy.crs as ccrs
-from cartopy.mpl import _MPL_36
+from cartopy.mpl import _MPL_37
 
 
 @pytest.mark.natural_earth
 @pytest.mark.mpl_image_compare(
-    filename="igh_land.png", tolerance=0.5 if _MPL_36 else 3.6)
+    filename="igh_land.png", tolerance=0.5 if _MPL_37 else 3.6)
```

Comment:
We did this [last time too](https://github.com/SciTools/cartopy/pull/2258/files#diff-18c2d8f7b2271b656b2822b5984e1e1c2ee1d768a243c1b47136ee435f98b583), based on failures in the min version test.  I'm starting to think maybe it's not the mpl version making the difference.  I don't think I want to start digging into that today though.